### ISSUE TITLE
Request: output resolution

### ISSUE THREAD

**Issue by Echolink50** [NextDiffusion/next-view#6]
Thanks for the great extension. It's very handy because I usually open blender for these tasks. I was wondering if it would be possible to add the ability to select the resolution for the output. Sometimes I want to scale 4K down to 1080p. Thanks again

**Comment by robertJene** [comment]
you could use the "extras" tab with the batch from directory option to upscale the images after a video was turned into images

then have NextDiffusion turn the upscaled images back into a video

but you need to be very familiar with upscaler support in Automatic1111, like what each model does, and the different options

**Comment by tonywhite11** [comment]
Yeah, my problem was that I didn't realize that the input frames had to be in the frame_0001 format and was trying to upload image sequences from other software. As for the upscale issue, I usually make a minute to a minute and a half videos and that's a lot of frames so using extras ans batching them that way took hours. I have Gigapixel for upscaling and it'll do thousands of frames in mutes but it adds a suffix and everyone knows renames file sequentially in windows is a nightmare and the online sources are iffy. So I wrote this 27 lines of code that would take my file from anywhere and rename them to frame_XXXX. That feature would be great with the extension. Here's the code I used. -

import os
import shutil

def rename_files(input_dir, output_dir, prefix):
    # Ensure output directory exists
    if not os.path.exists(output_dir):
        os.makedirs(output_dir)

    # Get list of files in the input directory
    files = os.listdir(input_dir)

    # Sort the files
    files.sort()

    # Loop over the files
    for i, filename in enumerate(files, start=1):
        # Create new filename
        new_filename = f"{prefix}_{str(i).zfill(4)}{os.path.splitext(filename)[1]}"

        # Copy and rename the file to the output directory
        shutil.copy(os.path.join(input_dir, filename), os.path.join(output_dir, new_filename))

# Usage
input_dir = 'Path_to_your_input_frames'
output_dir = 'Path_to_your_output_frames'
prefix = 'frame'
rename_files(input_dir, output_dir, prefix)

Great extension btw.
